src/cdk-experimental/ui-patterns/behaviors/list-selection/list-selection.ts

```diff
@@ -60,7 +60,14 @@ export class ListSelection<T extends ListSelectionItem<V>, V> {
 
     // TODO: Need to discuss when to drop this.
     this._anchor();
-    this.inputs.value.update(values => values.concat(item.value()));
+
+    const orderedValues = [];
```

Comment:
Are we sure this is the behavior we want? It means someone can't build a "pick your top 3" listbox that actually ranks the first, second, third place choices

Reply:
Hmm, interesting. I'm not sure how important it is to maintain the order of the source options. I believe it was Kristiyan who proposed this change. @crisbeto could you weigh in here?

Reply:
I think I'll leave it unsorted for now and wait for feedback from the community on this one. It's easier to add this feature in the future than it would be to remove it. I'm also not convinced we should add a hook for sorting since signals makes it trivial for devs to implement this by just using `computed`